### DISCUSSION TITLE
Order() for bash requests

### DISCUSSION THREAD

**Discussion by orisa** [supabase/supabase#8488] [Questions] [answered]
Good day. I'm currently using the bash API for my tables. Is it possible to order the response data(in ascending/descending) of requests? If so how do I do that. If not, is there any work around?

**Comment by GaryAustin1** [comment, marked as answer]
If you are going directly to the API then a good reference to use (beyond the basic API docs on the instance dashboard) is PostgREST docs.  https://postgrest.org/en/stable/api.html#ordering  Shows the URL format for order.

**Reply by orisa** [reply]
Thanks. I will have a look.
